libraries/lib-datahandler/src/main/thrift/vulnerabilities.thrift

```diff
@@ -42,13 +49,22 @@ struct Vulnerability{
     15: optional string priority,
     16: optional string priorityText,
     17: optional string action,
-    19: optional string impact,
+    19: optional map<string, string> impact,
     20: optional string legalNotice,
     21: optional set<string> assignedExtComponentIds,
     22: optional set<CVEReference> cveReferences,
     23: optional set<VendorAdvisory> vendorAdvisories,
     24: optional string extendedDescription,
     25: optional set<string> references,
+
+    //additional from CVE earch data
+    30: optional double cvss,
+    31: optional bool isSetCvss,
```

Comment:
No, it is unfortunately not enough, since the double value will always be initialized to 0 in the thrift object. And then you do not know whether this is the default 0 or an intentional 0. `isSetCvss` indicates whether the value was set in the CVE you got from the external source.